src/media/css/detail/more-info.styl

```diff
@@ -5,10 +5,13 @@
 @import '../lib';
 
 .app-more-metadata {
+        margin: 0 -12px;
     dl {
-        display: inline-block;
+        display: block;
         vertical-align: top;
-        width-calc('50% - 2px');
+        width: 50%;
+        float: left;
```

Comment:
If you leave this as `display: inline-block;` you shouldn't need to set it to `float: left;`.